dev/archery/archery/integration/tester_cpp.py

```diff
@@ -60,6 +60,10 @@ def _run(self, arrow_path=None, json_path=None, command='VALIDATE',
         if quirks:
             if "no_decimal_validate" in quirks:
                 cmd.append("--validate_decimals=false")
+            if "no_date64_validate" in quirks:
+                cmd.append("--validate_date64=false")
+            if "no_times_validate" in quirks:
+                cmd.append("--validate_times=false")
```

Comment:
Just a question - do all existing files have invalid times generated, or just some? Since it seems the current code path is correct, so presumably it was fixed a while back.

Reply:
There were 3 golden files that had invalid times resulting failure in the integration tests. So not all of them. However I think that all files did had invalid date64 value.